packages/forms/frontend/sirius-components-forms/src/propertysections/ChartWidgetPropertySection.tsx

```diff
@@ -37,11 +37,28 @@ export const ChartWidgetPropertySection: PropertySectionComponent<GQLChartWidget
   if (isBarChart(chart)) {
     chartComponent = (
       <div className={classes.chart}>
-        <BarChart chart={chart} />
+        <BarChart
+          chart={{
+            entries: chart.entries,
+            height: chart.height,
+            width: chart.width,
+            yAxisLabel: chart.yAxisLabel,
+            style: chart.style,
```

Comment:
Since you removed the label, you can revert that change too

Reply:
done